### PR TITLE
Elementwise simd: adding numerical tests

### PR DESCRIPTION
**Pull request by AlexandreEichenberger**
Adding numerical tests 

plus fix suggested by @sorenlassen about an unsafe debug printf.

### CODE REVIEW

**Review by sorenlassen — APPROVED** [review]
LGTM

File: test/numerical/TestElementwise.cpp [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  using namespace onnx_mlir;
+  using namespace onnx_mlir::test;
+
+  llvm::FileRemover remover(
+      onnx_mlir::getTargetFilename(SHARED_LIB_BASE.str(), onnx_mlir::EmitLib));
+
+  ModelLibBuilder::setRandomNumberGeneratorSeed("TEST_SEED");
+  setCompilerOption(OptionKind::CompilerOptLevel, "3");
+  llvm::cl::ParseCommandLineOptions(
+      argc, argv, "TestGemm\n", nullptr, "TEST_ARGS");
```
Comment (tungld) [inline test/numerical/TestElementwise.cpp:61]: Should this be `TestElementwise`?

File: src/Conversion/ONNXToKrnl/Math/Elementwise.cpp [hunk 2]
```diff
@@ -1025,7 +1021,7 @@ static LogicalResult getVariadicSimdCodeFullyFlattened(
   unsigned numArgs = op->getNumOperands();
 
   if (DEBUG)
-    fprintf(stderr, "SIMD code for variadic op %s\n", getOpName(op));
+    llvm::errs() << "SIMD code for variadic op " << op->getName() << "\n";
```
Comment (tungld) [inline src/Conversion/ONNXToKrnl/Math/Elementwise.cpp:1024]: Is there any particular reason to use `errs()` instead of `outs()`?

**Review by tungld — COMMENTED** [review]
@AlexandreEichenberger do you plan to enable this on Jenkins CI?

File: src/Conversion/ONNXToKrnl/Math/Elementwise.cpp (hunk 2 above)
Comment (AlexandreEichenberger) [reply src/Conversion/ONNXToKrnl/Math/Elementwise.cpp:1024]: yes, it is just a way for me to enable/disable simd for the ops that show up in a model.

**Review by tungld — APPROVED** [review]
LGTM.

File: test/numerical/TestElementwise.cpp (hunk 1 above)
Comment (AlexandreEichenberger) [reply test/numerical/TestElementwise.cpp:61]: thanks

**Review by tungld — APPROVED** [review]
LGTM.

**Pull request merged**